presto-verifier/pom.xml

```diff
@@ -108,12 +108,6 @@
             <artifactId>guava</artifactId>
         </dependency>
 
-        <dependency>
```

Comment:
Unrelated change?

Reply:
Got "duplicate dependency" error after adding findbugs to jdbc/pom.xml. Removing it fixed the error.